pkg/cmd/server/origin/asset.go

```diff
@@ -140,6 +140,10 @@ func (c *AssetConfig) buildHandler() (http.Handler, error) {
 	// Generated config.js can not be cached since it changes depending on startup options
 	handler = assets.GeneratedConfigHandler(config, handler)
 
+	// Handle extensions
+	handler = assets.ExtensionHandler(handler)
+	handler = assets.StandaloneExtensionHandler(handler)
```

Comment:
we probably want to rework this as a flat mux to avoid really deep chains:
```
/  -> gzip (
  /scripts/extensions.js -> js handler
  /styles/extensions.css -> css handler
  /extensions/ -> extensions handler
  /config.js -> generated config handler
  / -> cache(html5(assetfs handler))
)
```